examples/CMakeLists.txt

```diff
@@ -165,3 +165,65 @@ else()
             "IPC examples with UMF pool API are supported on Linux only - skipping"
     )
 endif()
+
+if(LINUX)
+    set(EXAMPLE_NAME umf_example_memspace_numa)
+
+    add_umf_executable(
+        NAME ${EXAMPLE_NAME}
+        SRCS memspace/memspace_numa.c
+        LIBS umf hwloc numa)
+
+    target_include_directories(
+        ${EXAMPLE_NAME} PRIVATE ${UMF_CMAKE_SOURCE_DIR}/src/utils
+                                ${UMF_CMAKE_SOURCE_DIR}/include)
+
+    target_link_directories(${EXAMPLE_NAME} PRIVATE ${LIBHWLOC_LIBRARY_DIRS})
+
+    add_test(
+        NAME ${EXAMPLE_NAME}
+        COMMAND ${EXAMPLE_NAME}
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+
+    set(EXAMPLE_NAME umf_example_memspace_hmat)
+
+    add_umf_executable(
+        NAME ${EXAMPLE_NAME}
+        SRCS memspace/memspace_hmat.c
+        LIBS umf hwloc numa)
+
+    target_include_directories(
+        ${EXAMPLE_NAME} PRIVATE ${UMF_CMAKE_SOURCE_DIR}/src/utils
+                                ${UMF_CMAKE_SOURCE_DIR}/include)
+
+    target_link_directories(${EXAMPLE_NAME} PRIVATE ${LIBHWLOC_LIBRARY_DIRS})
+
+    add_test(
```

Comment:
I guess `hmat` example should be only run in (selected) QEMU builds - you'd have to filter it out in CI level...?

Reply:
Done